Jenkinsfile

```diff
@@ -0,0 +1,88 @@
+pipeline {
```

Comment:
@JackLivio no, we use this for continuous integration of RC feature and would like to use such in develop. I will remove it form RC. And probably we will introduce it to develop with separate PR with continuous integration